### DISCUSSION TITLE
Getting a ChatIdInvalidError

### DISCUSSION THREAD

**Discussion by jerung3000** [aahnik/tgcf#98] [General]
Hi aahnik.

When I run yor tgcf it can forward message to group with 2 or 3 people in side.. But when I forward message to the channel that are more than 10 people..I get below error message.

_telethon.errors.rpcerrorlist.ChatIdInvalidError: Invalid object ID for a chat. Make sure to pass the right types, for instance making sure that the request is designed for chats (not channels/megagroups) or otherwise look for a different one more suited\nAn example working with a megagroup and AddChatUserRequest, it will fail because megagroups are channels. Use InviteToChannelRequest instead (caused by SendMediaRequest)_

How to set channel or megagroups into chat or how to enable message forward into channel?

**Comment by aahnik** [comment]
Hi @jerung3000,

tgcf can forward messages from any channel/group/chat to any channel/group/chat.

> But when I forward message to the channel that are more than 10 people..I get below error message.

Your reasoning of "more than 10 people" is **completely wrong. That is not a reason for any error.**

- I think you are doing something in the wrong way!
- Have you set the correct chat ids in the configuration?
- Does the channel/group exist? Have you access to the channel/group ?
- **I think you have used a wrong chat id**. You may directly use the username of the channel or group. But if it is a private one, then using the chat id is recommended. You can get proper chat ids from [userinfobot](https://telegram.me/userinfobot)
Please give more details. Show me your `tgcf.config.yml`, and also some screenshots of the channel or group that you are talking about.

_Make sure to star and watch the repo, and also join our telegram channel [tg_cf](https://telegram.me/tg_cf)_

**Reply by jerung3000** [reply]
hi aahnik,

Thank for quick response,

I already put the correct ID from the private group, and it works..As you told me there were something wrong with my ID 👍 

I found the way to get the channel ID with so simple 👍 

1-Go to https://web.telegram.org
2-Click on your channel
3-Look at the URL and find the part that looks like c12112121212_17878787878787878
4-Remove the underscore and after c12112121212
5-Remove the prefixed letter 12112121212
6-Prefix with a -100 so -10012112121212
7-That's your channel id.

Anyways, thanks aahnik.

**Reply by aahnik** [reply]
I recommend using the desktop app for telegram (available for Windows/Mac/Linux) or the Mobile app (Android/iOs)

The web app method you said may not be fully reliable in future.

So use userinfobot https://telegram.me/userinfobot to know chat ids.

**tgcf now supports using usernames/links/phone numbers in the [configuration file](https://github.com/aahnik/tgcf/wiki/How-to-configure-tgcf-%3F).**

Using chat id is no longer mandatory.
